src/frontend/components/PublicVideoDashboard/index.tsx

```diff
@@ -55,20 +57,23 @@ const PublicVideoDashboard = ({
                 </Box>
               )}
             </Box>
-            {video.xmpp && video.live_type === LiveModeType.JITSI && (
-              <Box
-                direction="row"
-                margin="small"
-                alignContent="center"
-                justify="center"
-              >
-                <JoinDiscussionAskButton />
-              </Box>
-            )}
+            {video.xmpp &&
```

Comment:
This condition statement starts to be complicated. What do you think to create a variable to store result of this test?

```ts
const isJitsiLiveNotPlaying = useMemo(() => 
  video.xmpp
  && video.live_type === LiveModeType.JITSI
  && ![liveState.STOPPING, liveState.PAUSED].includes(video.live_state)
, [video])


...
{ isJitsiLiveNotPlaying && <Box ... /> }
```

Reply:
Good idea.